docs/source/developer.rst

```diff
@@ -101,6 +101,30 @@ Impala (with UDFs)
 
       ci/impalamgr.py load --data --data-dir ibis-testing-data
 
+BigQuery
+^^^^^^^^
+
+Before you begin, you must have a `Google Cloud Platform project
+<https://cloud.google.com/docs/overview/#projects>`__ with billing set up the
```

Comment:
Did you mean `and the BigQuery API enabled`?

Reply:
Yes. Fixed in https://github.com/ibis-project/ibis/pull/1523/commits/14eb0ca170bc89693410b62b1ea1bda78329a647